app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="fr.free.nrw.commons">
```

Comment:
Refer to[ this comment](https://github.com/commons-app/apps-android-commons/pull/5220/files#r1183287416).